login/login.js

```diff
@@ -1,40 +1,68 @@
-const constants = require('../utils/constants')
+const StatusCodes = require('http-status-codes');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const moment = require('moment');
+const db = require('../queries/queries');
 
-const loginUser = (req, res) => {
-  if (!req.body) return res.sendStatus(400);
+const loginUser = async (req, res) => {
+  if (!req.body) {
+    return res.sendStatus(400);
```

Comment:
Could you please use Http Stattus codes package here ?

Reply:
fixed